src/commands/play.rs

```diff
@@ -21,15 +21,17 @@ pub async fn play(
     ctx: &Context,
     interaction: &mut ApplicationCommandInteraction,
 ) -> Result<(), SerenityError> {
-    _play(ctx, interaction, &PlayFlag::DEFAULT).await
+    _play(ctx, interaction, &PlayFlag::END).await
```

Comment:
Now that we are it, let's change the `_play` signature to take `flag: PlayFlag` instead of a reference (does not really make sense to pass a reference for an enum value)